addons/common/functions/fnc_isRemoteControlled.sqf

```diff
@@ -0,0 +1,22 @@
+#include "script_component.hpp"
```

Comment:
Function is missing `PREP` entry.

Reply:
The benefit of coding at 4AM, you are bound to forget things. 😅